src/modules/Core.js

```diff
@@ -178,9 +178,16 @@ export default class Core {
       i: [],
     }
 
+    let chartType = cnf.chart.type !== 'undefined' ? cnf.chart.type : 'line'
```

Comment:
undefined as a string should be checked as `typeof cnf.chart.type !== 'undefined'`